dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/template/components/MultiSelectComponentFactory.kt

```diff
@@ -21,7 +21,6 @@ class MultiSelectComponentFactory(@Autowired val templateDiagnostic: TemplateDia
         utils: ComponentGenerationUtils,
         componentGroup: ComponentGroupApi,
     ): ComponentBase {
-        templateDiagnostic.optionsNotUsed(row)
```

Comment:
Why was this even there?

Reply:
That must have been a copy-paste-leftover by me. 
Of course it does not make sense to assert that the options column is empty for a Multi-Select field row in the excel.
You need to paste values there to pass the options to the field.

Reply:
And we just overllooked this because we used the indevpavedroadframework? With the pavedroadframework we would have noticed that right?

Reply:
Yes I think so.  I think the PavedRoadFramework would have stopped the run at this point.
The InDevelopment... just logs warnings and continues.